app/static/js/zseries_recipe.js

```diff
@@ -182,11 +185,22 @@ function calculate_hop_timing(data, provided_table = undefined) {
         var rows = provided_table.getRows();
         var adjunctSteps = rows.filter(row => row.getData().location.indexOf("Adjunct") == 0);
 
-        var cumulative_hop_time = 0;
+        var cumulative_hop_time = {
+            "Adjunct1": 0,
+            "Adjunct2": 0,
+            "Adjunct3": 0,
+            "Adjunct4": 0
+        };
         adjunctSteps.slice().reverse().forEach(adjunctRow => {
             var row_data = adjunctRow.getData();
-            cumulative_hop_time += row_data.step_time;
-            adjunctRow.update({ "hop_time": cumulative_hop_time });
+            for (x in cumulative_hop_time) {
+                if (row_data.location <= "Adjunct" + x) {
+                    cumulative_hop_time[x] += row_data.step_time
+                }
+            }
+
+            // cumulative_hop_time += row_data.step_time;
+            adjunctRow.update({ "hop_time": cumulative_hop_time[row_data.location] });
```

Comment:
![image](https://user-images.githubusercontent.com/2158627/100530498-da3f5e80-31c0-11eb-83ec-ca2466b74b55.png)

Reply:
The steps are additive if you have multiple as the row calculations are taking place from the bottom of the table to the top (reversed order). This way from the first of your adjunct steps using the same location you see the total time for that location vs later steps that just show "remaining time" at that location (useful if you might be adding a 5th, 6th, 7th hop step...)